web/app/reset-new/page.tsx

```diff
@@ -3,11 +3,9 @@ import ResetNew from "@/components/Authentication/ResetNew";
 import ResetBackground from "@/components/Authentication/ResetBackground";
 
 const Reset: React.FC = () => {
-  const handleAction = () => {};
-
   return (
     <div>
-      <ResetBackground description="Reset password" card={ResetNew} action={handleAction} />
+      <ResetBackground description="Reset password" Card={ResetNew} />
```

Comment:
Can you check if this breaks the code? What Raph's PR introduced yesterday goes against this I think.

Reply:
No please it works perfectly